### ISSUE TITLE
mwoffliner 1.9.8 docker image seems broken

### ISSUE THREAD

**Issue by rgaudin** [openzim/mwoffliner#975]
All runs with `1.9.8` on the zimfarm fails with:

```
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '../../src/stores'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/tmp/mwoffliner/lib/util/redirects.js:13:18)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/tmp/mwoffliner/lib/mwoffliner.lib.js:47:21)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/tmp/mwoffliner/lib/cli.js:34:36)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
```

**Comment by rgaudin** [comment]
https://farm.openzim.org/tasks/5d720f591d1619754c6a4da3

**Comment by kelson42** [comment]
@ISNIT0 I thought this problem was fixed?

**Comment by ISNIT0** [comment]
Gah! It's the auto-import stuff. I'll sort it now

**Comment by ISNIT0** [comment]
Fixed in 1.9.10
